Dockerfile

```diff
@@ -14,7 +14,10 @@ ARG BASE="fpm-alpine"
 
 # full kimai source
 FROM alpine:3.12 AS git-dev
-ARG KIMAI="1.8"
+ARG KIMAI="1.11.1"
+# I need to do this check somewhere, we discard all but the checkout so doing here doesn't hurt
+ADD test-kimai-version.sh /test-kimai-version.sh
```

Comment:
Just one remark: could it be necessary to run the checks / initialization at this point in the future? then maybe the file could be called a bit more generic like `init.sh` or `validate-environemnt.sh.`